src/views/product-tutorials-view/components/product-view-content/components/branded-callout/branded-callout.module.css

```diff
@@ -35,7 +35,8 @@
 .productPattern {
   display: none;
 
-  @media (--dev-dot-tablet-up) {
+  /** Using a custom mq here: The pattern looked 'broken' with the sidebar / sidecar layout at sizes <1200px */
+  @media only screen and (min-width: 1150px) {
```

Comment:
🙌 I think this is a great short-term fix.
💭 We might be able to avoid having to make it though, since it looks like #442 will address this for us! 💯

Reply:
Good call, perhaps I'll defer this until #442 is merged

Reply:
K I'm going to revert this as it seems #442 fixes the issue